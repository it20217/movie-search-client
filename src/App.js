import React from 'react';
import './App.css';
import Header from "./components/Header"

function App() {
  return (
    <div>
      <Header/>
      <h1 className="text-3xl font-bold underline">
      Movie app!
    </h1>
    </div>
  );
}

export default App;
