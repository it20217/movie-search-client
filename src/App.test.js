import React from 'react';
import { render, screen } from '@testing-library/react';
/** Snapshots rendering */
import renderer from "react-test-renderer";
import App from './App';


describe("App page component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<App/>).toJSON();
    expect(component).toMatchSnapshot();
  })
})
