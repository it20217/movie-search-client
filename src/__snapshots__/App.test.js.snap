// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App page component renders correctly 1`] = `
<div
  className="container-wrapper"
>
  <div
    className="header"
  >
    <div
      className="app-logo"
      onClick={[Function]}
    >
      <img
        alt="Logo"
        src="cinema-154392__340.webp"
      />
    </div>
    <div
      className="app-name text-gray-800"
    >
      Movie app!
    </div>
    <div
      className="search"
    >
      <div
        className="form-wrapper"
        style={
          Object {
            "marginTop": undefined,
          }
        }
      >
        <div
          className="search-input"
        >
          <input
            className="search-input"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Spiderman: No Way Home"
            style={
              Object {
                "borderColor": "#273461",
              }
            }
            type="search"
            value=""
          />
        </div>
        <div
          className="btn-container"
        >
          <button
            className="form-btn"
            onClick={[Function]}
            style={
              Object {
                "Color": "#273461",
                "borderColor": "#273461",
              }
            }
            type="button"
          >
            <svg
              fill="none"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M14 5l7 7m0 0l-7 7m7-7H3"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
    <div
      className="menu-icon-div"
      onClick={[Function]}
    >
      <svg
        fill="none"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 6h16M4 12h16M4 18h16"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="2"
        />
      </svg>
    </div>
  </div>
  <div
    className="Navbar movedIn"
  >
    <div
      className="flex bg-gray-200"
    >
      <aside
        className="flex flex-col items-center bg-white text-gray-700 shadow "
      >
        <ul>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex-col flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                <svg
                  className="h-6 w-6"
                  fill="none"
                  stroke="currentColor"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6"
                    strokeLinecap="round"
                    strokeLinejoin="round"
                  />
                </svg>
              </a>
            </div>
          </li>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex-col flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/contact-us"
                onClick={[Function]}
              >
                <svg
                  className="h-6 w-6"
                  fill="none"
                  stroke="currentColor"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M16 12a4 4 0 10-8 0 4 4 0 008 0zm0 0v1.5a2.5 2.5 0 005 0V12a9 9 0 10-9 9m4.5-1.206a8.959 8.959 0 01-4.5 1.207"
                    strokeLinecap="round"
                    strokeLinejoin="round"
                  />
                </svg>
              </a>
            </div>
          </li>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/bookmarks"
                onClick={[Function]}
              >
                <svg
                  className="h-5 w-5"
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M19 21l-7-5-7 5V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2z"
                  />
                </svg>
              </a>
            </div>
          </li>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/basket"
                onClick={[Function]}
              >
                <svg
                  className="h-5 w-5"
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="9"
                    cy="21"
                    r="1"
                  />
                  <circle
                    cx="20"
                    cy="21"
                    r="1"
                  />
                  <path
                    d="M1 1h4l2.68 13.39a2 2 0 0 0 2 1.61h9.72a2 2 0 0 0 2-1.61L23 6H6"
                  />
                </svg>
              </a>
            </div>
          </li>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/settings"
                onClick={[Function]}
              >
                <svg
                  className="h-5 w-5"
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="3"
                  />
                  <path
                    d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
                  />
                </svg>
              </a>
            </div>
          </li>
          <li
            className="hover:bg-gray-100"
          >
            <div
              className="h-16 px-6 flex flex justify-center items-center w-full focus:text-orange-500"
            >
              <a
                href="/notifications"
                onClick={[Function]}
              >
                <svg
                  className="h-5 w-5"
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M18 8A6 6 0 0 0 6 8c0 7-3 9-3 9h18s-3-2-3-9"
                  />
                  <path
                    d="M13.73 21a2 2 0 0 1-3.46 0"
                  />
                </svg>
              </a>
            </div>
          </li>
        </ul>
        <div
          className="mt-auto h-16 flex items-center w-full"
        >
          <button
            className="h-16 w-10 mx-auto flex flex justify-center items-center w-full focus:text-orange-500 hover:bg-red-200 focus:outline-none"
          >
            <svg
              className="h-5 w-5 text-red-700"
              fill="none"
              height="24"
              stroke="currentColor"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"
              />
              <polyline
                points="16 17 21 12 16 7"
              />
              <line
                x1="21"
                x2="9"
                y1="12"
                y2="12"
              />
            </svg>
          </button>
        </div>
      </aside>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="home-container"
    >
      <div
        className="image-container"
      />
      <div
        className="image-container"
      >
        <img
          alt="film-projector"
          src="stock-vector-set-of-popcorn-d-glasses-cinematograph-tape-cinema-ticket-drawn-vector-illustration-realistic-635850815.jpg"
        />
      </div>
      <div
        className="search"
      >
        <div
          className="form-wrapper"
          style={
            Object {
              "marginTop": 80,
            }
          }
        >
          <div
            className="search-input"
          >
            <input
              className="search-input"
              onChange={[Function]}
              onKeyDown={[Function]}
              placeholder="Spiderman: No Way Home"
              style={
                Object {
                  "borderColor": "#273461",
                }
              }
              type="search"
              value=""
            />
          </div>
          <div
            className="btn-container"
          >
            <button
              className="form-btn"
              onClick={[Function]}
              style={
                Object {
                  "Color": "#273461",
                  "borderColor": "#273461",
                }
              }
              type="button"
            >
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M14 5l7 7m0 0l-7 7m7-7H3"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="wrapper"
  >
    <div
      className="footer"
    >
      <div
        className="logo"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt="Logo"
            className="pic"
            src="cinema-154392__340.webp"
          />
        </a>
      </div>
      <div
        className="links"
      >
        <div>
          <a
            className="link"
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
          <br />
          <br />
          <a
            className="link"
            href="/read-more"
            onClick={[Function]}
          >
            Read More 
          </a>
          <br />
          <br />
          <a
            className="link"
            href="/contact-us"
            onClick={[Function]}
          >
            Contact us 
          </a>
          <br />
        </div>
      </div>
      <div
        className="side"
      >
        <address>
          <span>
            Movie app!
          </span>
          <br />
          email@email.com
          <br />
          0123456789
          <br />
          @movie_app!
        </address>
      </div>
    </div>
    <div
      className="project"
    >
      <hr />
      <hr />
      <br />
      Project created by 
      <span>
        Jaroslaw Polowy
      </span>
       and 
      <span>
        Igor Tokarev
      </span>
       © 2022 
      <a
        href="https://www.ncirl.ie/"
        rel="noreferrer"
        target="_blank"
      >
        NCI
      </a>
    </div>
  </footer>
</div>
`;
